tests/run-multitests.py

```diff
@@ -323,6 +324,18 @@ def run_test_on_instances(test_file, num_instances, instances):
     return error, skip, output_str
 
 
+def print_diff(a, b):
+    a_fd, a_path = tempfile.mkstemp(text=True)
+    b_fd, b_path = tempfile.mkstemp(text=True)
+    os.write(a_fd, a.encode())
+    os.write(b_fd, b.encode())
+    os.close(a_fd)
+    os.close(b_fd)
+    subprocess.run(["diff", a_path, b_path])
```

Comment:
see run-tests: to work on windows this needs to use `DIFF = os.getenv('MICROPY_DIFF', 'diff -u')`